### PR TITLE
CPAAS1238 : Keep destination when applying override

### PR DESCRIPTION
**Pull request by rnc**

### CODE REVIEW

File: docs/handbook/overrides.rst [hunk 1]
```diff
@@ -212,3 +212,68 @@ Example
...
+
+If the original contains a target definition and the override does **not** specify a target then the original value will
+be maintained.
+
+Examples
+^^^^^^^^
+
+1. Maintain destination with plain override with new target
+
+    .. code-block:: py
```
Comment (goldmann) [inline docs/handbook/overrides.rst:233]: Why not use `yaml` here as well?

File: docs/handbook/overrides.rst [hunk 2]
```diff
@@ -212,3 +212,68 @@ Example
...
+
+        name: 'bar.jar'
+        dest: '/tmp/destination/'
+        md5: 234234234234
+        target: 'bar2222.jar'
+
+
+2. Maintain destination and target with plain override:
+
+    .. code-block:: py
```
Comment (goldmann) [inline docs/handbook/overrides.rst:259]: Same.

**Comment by goldmann** [comment]
What happens to the `description` property? Shouldn't it be covered the same way?

File: docs/handbook/overrides.rst (hunk 1 above)
Comment (rnc) [reply docs/handbook/overrides.rst:233]: Um. typo? Will change :-)

**Comment by rnc** [comment]
> What happens to the `description` property? Shouldn't it be covered the same way?

Added coverage for that as well.

**Review by goldmann — APPROVED** [review]

**Pull request merged**